ServerCore/wwwroot/js/pd2022/help.js

```diff
@@ -13,10 +13,10 @@ window.addEventListener("load", function (event) {
 	let helpspan = document.createElement("span");
 	helpspan.classList.add("help-link");
 	let helplink = document.createElement("button");
-	helplink.textContent = "click here";
+	helplink.textContent = "click here to ask the author for help";
 	helplink.classList.add("help-button");
 	helplink.onclick = clicked;
-	helpspan.append(" (", helplink, " to ask the author for help)");
+	helpspan.append(" (", helplink, ")");
```

Comment:
Will this cause the parentheses to ever break separately from the button if the author name(s) is/are long? Maybe remove the parentheses altogether?